lisa/sut_orchestrator/azure/tools.py

```diff
@@ -55,3 +57,51 @@ def get_generation(self) -> str:
         else:
             generation = "1"
         return generation
+
+
+class LisDriver(Tool):
+    """
+    This is a virtual tool to detect/install LIS drivers.
+    """
+
+    @property
+    def command(self) -> str:
+        return "modinfo hv_vmbus"
+
+    def _check_exists(self) -> bool:
+        if not self.node.os.is_posix:
```

Comment:
If it doesn't support install, override the can_install.

Reply:
Didnt understand this. Do you mean class LisDriver doesn't support install?

Reply:
If return false in `_check_exists`, the LISA will try to check `can_install`. If it returns True, LISA will call `install` to install the tool. So, make sure LIS is not installed on unsupported oses.